notebooks/SVD_tools.py

```diff
@@ -0,0 +1,87 @@
+## Modified from Gretchen's CLM_SVD notebook
+## functions used in SVD used here 
+
+import numpy as np
+import matplotlib.pyplot as plt
+import netCDF4 as nc
+from scipy import stats
+
+
+def decompose(ts_anomaly):
```

Comment:
I think it would be worth adding a few [numpy-style docstrings](https://numpydoc.readthedocs.io/en/latest/format.html) to the functions in this module. Taken out of context, its hard for me to track what input/output shapes I should expect for the various variables.